### ISSUE TITLE
Streamline the creation of utility classes

### ISSUE THREAD

**Issue by Luchian** [pixeltrove/spark-css#1]
Adding or editing utility classes manually is cumbersome and error-prone. A mixin that receives values individually, as a list, or via a map and uses them to generate the required classes would ease development.

**Comment by Luchian** [comment]
The mixin should be able to generate utility classes from a single value or a list of values
```
$display-values: block, flex, none;
```
or from a map that specifies values for non-responsive classes and those required within particular viewport sizes
```
$margin-values: (
  standard: (
    regular: 1rem
  ),
  small-up: (
    regular: 1rem
  )
);
```

**Comment by Luchian** [comment]
Using the mixin should be straightforward and easy to understand. Including it in the appropriate utility file along with the desired property, values, and an optional alias would work best like so `@include utility(display, $display-values, d);`.

**Comment by Luchian** [comment]
Implemented in 0.5.0.
